docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.3"
+services:
+  automod:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    environment:
+      - LEMMY_DOMAIN=
+      - LEMMY_USERNAME=
+      - LEMMY_PASSWORD=
+      - USE_SQLITE=1
+      - THREATIVORE_ADMIN_URL=
+    restart: always
+    ports:
+      - 8000:8080
+    volumes:
+      - ./volumes/automoderator:/usr/src/app/database
```

Comment:
The default sqlite location is not this path. Where did you get this from?

Reply:
Oh crap, this was leftover from a copy of something else.  Where does it save it by default? I'll update it to that

Reply:
I took a stab at it, but I could use a second pair of eyes on it.  I can verify 100% the postgres version works, but I don't have a test environment for the sqlite version.  Not to shlep it off, but that would be a great test for me, to see if you can spin up the container with sqlite and tell me it works (and that it persists after running it once).  It should literally be the same as the python version running it, you should be able to just point your volume to the directory your .db is in.